Source/ProgressionSystemRuntime/Private/Data/PSWorldSubsystem.cpp

```diff
@@ -329,6 +329,25 @@ void UPSWorldSubsystem::OnTakeActorsFromPoolCompleted(const TArray<FPoolObjectDa
 	}
 }
 
+// Returns spot component by the player tag, returns null if spot is not found
+UPSSpotComponent* UPSWorldSubsystem::GetCurrentSpot() const
+{
+	FPlayerTag PlayerTag = UMyBlueprintFunctionLibrary::GetLocalPlayerCharacter()->GetPlayerTag();
```

Comment:
It will crash the game if character is not found: never access pointers without verifying the pointer GetLocalPlayerCharacter()->

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/40/commits/5fa86f12a6a382a9c22cf7824d222cfd0a4f5a58